packages/contracts/src/dollar/facets/StakingFormulasFacet.sol

```diff
@@ -1,10 +1,11 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.19;
+pragma solidity ^0.8.18;
```

Comment:
Why was this and some of the other smart contracts downgraded?

Reply:
> Why was this and some of the other smart contracts downgraded?

oh my bad there was like this errors about incorrect solidity version to check if i can downgrade and fix them i tried doing it but seems like i forget to make some of them to 0.8.19 i will fix that